Examples and Compatibility for dSPACE tools

MBlesken opened the pull request:
@chrbertsch would you please review and approve this PR?

chrbertsch reviewed (approved):
Now the link works. 
And I just saw that the information on the validation tools is included in the README in each individaul zip file. (Perhaps you could add this information on the top level outside the zip files).

MBlesken commented:
@chrbertsch please check https://www.dspace.com/go/FMI-Compatibility again. We added the HW products you asked for and removed the obsolete column, too.
